package.json

```diff
@@ -31,7 +31,7 @@
     "logger-sharelatex": "^1.6.0",
     "metrics-sharelatex": "^2.1.1",
     "redis-sharelatex": "git+https://github.com/sharelatex/redis-sharelatex.git#v1.0.6",
-    "request": "~2.34.0",
+    "request": "^2.34.0",
```

Comment:
should this not read `2.88.0`?

Reply:
^2.34 will update to the latest before 3.0, so it actually gets 2.88 (which is the latest)
(~2.34.0 only updated to 2.34.x)
I can change it if it's clearer.

Reply:
I think we should change it as it is quite a few versions behind and people often look at the package.json to see what is installed.

Reply:
:+1: done